exercises/exercise_010_opaque_type_aliases/README.md

```diff
@@ -63,5 +63,8 @@ this exercise. Specifically we will convert the last of these type aliases,
   member of ...` by adding an extension method, and then continue with the other
   type of error.
 
-- Once all the compilation errors are fixed, verify that the application
-  continues to work correctly
+- Once all the compilation errors are fixed continues to work correctly, run
```

Comment:
Once all the compilation errors are fixed, ~~continues to work correctly,~~ run the provided tests